config/routes.rb

```diff
@@ -6,11 +6,11 @@
 
   namespace :admins do
```

Comment:
O padrão que o time concordou em usar foi no singular, então seria bom trocar para não haver conflito com a main
```suggestion
  namespace :admin do
```